wiki/Tournaments/OWC/2023/en.md

```diff
@@ -144,12 +198,78 @@ The complete sign-up list can be found [here](https://gist.github.com/LeoFLT/82e
+| #24 | ::{ flag=UA }:: Ukraine | 7.107 | 2,397,044 |
+| #25 | ::{ flag=TR }:: Turkey | 6.982 | 2,366,035 |
+| #26 | ::{ flag=TW }:: Taiwan | 6.837 | 2,318,130 |
+| #27 | ::{ flag=ES }:: Spain | 6.773 | 2,264,142 |
+| #28 | ::{ flag=MX }:: Mexico | 6.656 | 2,237,655 |
+| #29 | ::{ flag=NL }:: Netherlands | 6.621 | 2,201,197 |
+| #30 | ::{ flag=TH }:: Thailand | 6.565 | 2,193,178 |
+| #31 | ::{ flag=AR }:: Argentina | 6.309 | 2,140,036 |
+| #32 | ::{ flag=KZ }:: Kazakhstan | 6.14 | 2,086,765 |
+| #33 | ::{ flag=MY }:: Malaysia | 6.031 | 2,041,506 |
+| #34 | ::{ flag=IL }:: Israel | 5.79 | 1,930,594 |
+| #35 | ::{ flag=DK }:: Denmark | 5.764 | 1,923,570 |
+| #36 | ::{ flag=HU }:: Hungary | 5.618 | 1,906,952 |
+| #37 | ::{ flag=BE }:: Belgium | 5.584 | 1,880,677 |
+| #38 | ::{ flag=UY }:: Uruguay | 5.515 | 1,873,738 |
+| #39 | ::{ flag=PT }:: Portugal | 5.495 | 1,855,838 |
+| #40 | ::{ flag=BY }:: Belarus | 5.431 | 1,807,011 |
+| #41 | ::{ flag=EE }:: Estonia | 5.18 | 1,744,546 |
+| #42 | ::{ flag=VN }:: Vietnam | 5.169 | 1,745,088 |
+| #43 | ::{ flag=AT }:: Austria | 5.155 | 1,742,198 |
+| #44 | ::{ flag=CZ }:: Czechia | 5.074 | 1,729,518 |
+| #45 | ::{ flag=SA }:: Saudi Arabia | 4.967 | 1,668,352 |
+| #46 | ::{ flag=LV }:: Latvia | 4.94 | 1,655,513 |
+| #47 | ::{ flag=LT }:: Lithuania | 4.558 | 1,537,059 |
+| #48 | ::{ flag=KH }:: Cambodia | 4.166 | 1,432,747 |
+| #49 | ::{ flag=AE }:: United Arab Emirates | 4.101 | 1,420,705 |
+| #50 | ::{ flag=CO }:: Colombia | 4.053 | 1,353,254 |
+| #51 | ::{ flag=MN }:: Mongolia | 4.003 | 1,359,885 |
+| #52 | ::{ flag=PA }:: Panama | 3.398 | 1,145,791 |
+| #53 | ::{ flag=SK }:: Slovakia | 3.362 | 1,114,713 |
+| #54 | ::{ flag=ZA }:: South Africa | 3.349 | 1,141,540 |
+| #55 | ::{ flag=CH }:: Switzerland | 3.199 | 1,068,495 |
+| #56 | ::{ flag=HR }:: Croatia | 2.892 | 0,969,311 |
+| #57 | ::{ flag=IN }:: India | 2.69 | 0,892,297 |
+
 ## Ruleset
 
 ### Tournament rules
 
 1. The osu! World Cup is a country-based 4 versus 4 double-elimination team tournament, played on the osu! game mode.
-   - Players from countries or territories (as seen on their profile flag) that are under a protectorate or sovereign state may choose to play for that country **if** the country listed on their profile is unable to field enough players to form a valid team. Refer to [this article](https://wikipedia.org/wiki/Dependent_territory#Lists_of_dependent_territories) for an overview of the criteria adopted.
+   - Players from countries or territories (as seen on their profile flag) that are under a protectorate or sovereign state may choose to play for that country **if** the country listed on their profile is unable to field enough players to form a valid team. Refer to [this article](https://wikipedia.org/wiki/Dependent_territory#Lists_of_dependent_territories) for an overview of the criteria adopted. Players that have participated under another country previously may continue to do so.
```

Comment:
just want to clarify, does this added part apply to *all players* or just the players being talked about on this line so far?

Reply:
actually the wording is fairly clear rereading it again, nvm.